### PR TITLE
Feature/stat tracker dc

### PR DESCRIPTION
**Pull request by DChavez18**
Started Stat_tracker_spec
Test for exists
test for highest_total_score method
Created StatTracker class and initialization method and helper methods in order to create instances of teams and games
Created highest total score method (We may want to refactor)

### CODE REVIEW

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    stat_tracker.create_teams(teams_path)
+    stat_tracker
+  end
+
+  def highest_total_score
+    highest_score = 0
+
+    @games.each do |game|
+      total_score = game.away_goals + game.home_goals
+      highest_score = total_score if total_score > highest_score
```
Comment (dani-wilson) [inline lib/stat_tracker.rb:40]: Nicely done here 😄

**Review by dani-wilson — APPROVED** [review]
Looks good, Derek. I like the consistency of spacing & style!

File: lib/stat_tracker.rb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    stat_tracker.create_games(game_path)
+    stat_tracker.create_teams(teams_path)
+    stat_tracker
+  end
+
+  def highest_total_score
+    highest_score = 0
+
+    @games.each do |game|
+      total_score = game.away_goals + game.home_goals
```
Comment (algibson1) [inline lib/stat_tracker.rb:39]: Would max_by be a better enumerable for this?

**Comment by GVarenhorst** [comment]
Looks good.

File: lib/stat_tracker.rb (hunk 2 above)
Comment (algibson1) [reply lib/stat_tracker.rb:39]: ^^Apparently this comment was "pending" this whole time and not visible to anyone! I'm figuring out how to properly use the review functions, thanks for bearing with me!!

**Pull request merged**